generators/app/templates/boilerplate/gulp/tasks/test.js

```diff
@@ -1,14 +1,16 @@
+/*eslint quotes: ["error", "single"]*/
+// Related to: https://github.com/Microsoft/TypeScript/issues/13270'
 'use strict';
 
-var gulp = require('gulp');
-var browserOpen = require('gulp-open');
-var istanbul = require('gulp-istanbul');
-var mocha = require('gulp-mocha');
-var nsp = require('gulp-nsp');
+const gulp = require('gulp');
```

Comment:
I also tried to go through our existing JS files to ensure we are following ES6 contentions instead of ES5